sys/fuchsia/profile.txt

```diff
@@ -0,0 +1,16 @@
+# Copyright 2022 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+# See https://cs.opensource.google/fuchsia/fuchsia/+/main:zircon/vdso/profile.fidl
+
+include <zircon/syscalls.h>
+
+resource zx_root[zx_handle]
+
+zx_profile_info {
+    flags             int32
+    priority          int32
+    cpu_affinity_mask array[int64, 8]
+}
+
+zx_profile_create(root_job ptr[in, zx_root], options int32, profile ptr[in, zx_profile_info], out_handle ptr[out, zx_root])
```

Comment:
I think `root_job` should be `root_job zx_job`, not `ptr[zx_root]`. (`zx_job` is already defined for us in jobs.txt.) And then we need to define a `zx_profile` (as `resource zx_profile[zx_handle]`).

Reply:
Thanks!
`zx_profile` is never used as input anywhere, so I added it but commented out.